sklearn/utils/tests/test_sparsefuncs.py

```diff
@@ -85,6 +85,61 @@ def test_mean_variance_axis1():
             assert_array_almost_equal(X_vars, np.var(X_test, axis=0))
 
 
+@pytest.mark.parametrize('axis', (0, 1))
+def test_mean_variance_with_ddof(axis):
+    X, _ = make_classification(5, 4, random_state=0)
+    # Sparsify the array a little bit
+    X[[0, 2, 4], [0, 1, 3]] = 0
+
```

Comment:
If you want to generate a random sparse matrix youcan use `scipy.sparse.random`

Reply:
This is kind of pointless since we also require the dense matrix anyways when we compare results against numpy. Doing it this way seems more explicit to me.